activate.sh

```diff
@@ -46,3 +46,7 @@ if ! [ -f /root/.ssh/id_rsa ]; then
 fi
 [ -d /var/lib/misc/ssh-public-key  ] || mkdir -p /var/lib/misc/ssh-public-key
 cp /root/.ssh/id_rsa.pub /var/lib/misc/ssh-public-key
+
+# Create MariaDB system user, bsc#1034885
+groupadd -g 331 mysql
+useradd -r -u 331 -g mysql -c "MySQL database admin" -d /var/lib/mysql -s /bin/false mysql
```

Comment:
You better use uid 60 for mysql user, which is the one used by mariadb user:

See: https://build.suse.de/package/view_file/SUSE:SLE-12-SP2:Update/mariadb/mariadb.spec?expand=1

You can see in the pre section this:

getent group mysql >/dev/null || groupadd -r mysql
getent passwd mysql >/dev/null || useradd -r -o -g mysql -u 60 -c "MySQL database admin" \
                  -s /bin/false -d %{_localstatedir}/lib/mysql mysql
usermod -g mysql -s /bin/false mysql